### ISSUE TITLE
Update readme to reference python-wrapper branch

### ISSUE THREAD

**Issue by faoiseamh** [mapillary/OpenSfM#50]
The install instructions for OpenGV are missing a step to switch to the python-wrapper branch which has the python bindings

**Comment by geosky** [comment]
but make install opengv fail, see the new issue #51

**Comment by faoiseamh** [comment]
I'll submit a PR for this later

**Comment by paulinus** [comment]
Perfect, thanks!
We have a PR pending on the official OpenGV repository https://github.com/laurentkneip/opengv/pull/19. We need the better instructions you are proposing until the PR gets accepted.

**Comment by faoiseamh** [comment]
Resolved in #53
